### ISSUE TITLE
[Bug] pip Installation - Failed to build ecos scs - error

### ISSUE THREAD

**Issue by nicdan** [GamestonkTerminal/GamestonkTerminal#955]
**Describe the bug**
Running "pip install -r requirements.txt" it runs into " ERROR: Failed building wheel for ecos
  Running setup.py clean for ecos
  Building wheel for scs (pyproject.toml) ... error" 
which continues and ends with the error in the title.

**To Reproduce**
pip install -r requirements.txt

**Screenshots**
![error1](https://user-images.githubusercontent.com/1263622/142035300-a64be189-c564-449e-b4b2-612be8ad7a28.jpg)
![error2](https://user-images.githubusercontent.com/1263622/142035318-4afb18e9-2f6f-49c4-b5b3-9a3e4fdeabd6.jpg)
![error3](https://user-images.githubusercontent.com/1263622/142035335-8c292730-387d-4821-908b-9fdc0e2bf0f4.jpg)

**Desktop (please complete the following information):**

- OS: Win10 Pro
- Python version 3.8.12

**Additional context**
Add any other information that you think could be useful for us
installing through poetry also fails (it starts with "The virtual environment found in d:\anaconda3\envs\gst seems to be broken.") and then proceeds to fix it (every time I attempt, even on consecutive tries) and ends with
  
ValueError

  File \C:\Users\dan\AppData\Local\pypoetry\Cache\artifacts\be\98\c7\69fe6fea7a59659af1c6260899226129565330b1e07c9c5b3769be76bf\six-1.15.0-py2.py3-none-any.whl does not exist

  at d:\anaconda3\envs\gst\lib\site-packages\poetry\core\packages\file_dependency.py:40 in __init__
       36│             except FileNotFoundError:
       37│                 raise ValueError("Directory {} does not exist".format(self._path))
       38│
       39│         if not self._full_path.exists():
    →  40│             raise ValueError("File {} does not exist".format(self._path))
       41│
       42│         if self._full_path.is_dir():
       43│             raise ValueError("{} is a directory, expected a file".format(self._path))
       44│

**Comment by jmaslek** [comment]
That is usually related to an issue with C++.  Check out the troubleshooting guide:

https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/TROUBLESHOOT.md#microsoft-visual-v++

**Comment by nicdan** [comment]
I've had this problem since yesterday, and I did the C++ update, as well as everything in the poetry section, I'll do it again now and come back to report.

**Comment by nicdan** [comment]
Nope, it didn't work, exact same messages with both pip and poetry

**Comment by nicdan** [comment]
The solution was: nuke original env and create a new one, download Build Tools 2019!! (not the latest!). Still had errors with poetry, so still installed via pip. Pip still had errors while uninstalling some stuff after installation (no screenshots, sorry, I've done it a few hours ago on the run so I don't know what it was uninstalling), but played with the terminal for about an hour now and it works as expected (I had it on a different pc since may). Fantastic job guys! Thank you
TL;DR suspect compatibility issues, try earlier versions of build tools

**Comment by DidierRLopes** [comment]
We'll reflect this in our troubleshooting. Thank you @nicdan !
